js/Source/Delegators/Delegator.InputMirror.js

```diff
@@ -34,9 +39,17 @@ Delegator.register(['change','keyup'], {
         var targetElement = element.getElement(target.selector);
         if (!targetElement) api.warn('Unable to find element for inputMirror selector: '+target.selector);
         if (targetElement && targetElement != element){
-          targetElement.set(target.property || 'value', element.get('value'));
+          var value = element.get('value');
+          if (api.get('stripHTML')){
+            value = value.replace(/</g, '&#60;').replace(/>/g, '&#62;').replace(/\//g, '&#47;');
```

Comment:
from using it, it appears that i do need to replace these. if i don't, you can close the tag that this value is appearing in.